src/css/base.scss

```diff
@@ -1,6 +1,56 @@
-/* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: pink;
+// COLORS
 
-body {
-  background-color: $primary-background;
+$dark-gray: #4d4e4f;
```

Comment:
If we have time to circle back would we want to call the variables more like the general location of the colors rather then the color itself - in the Sass lesson it was "header" or background" as the variable. May not make sense for this project though.